default.nix

```diff
@@ -35,16 +42,30 @@
     cargo-nextest = fancy.rustOverrides super.cargo-nextest;
     csview = fancy.rustOverrides super.csview;
     just = fancy.rustOverrides super.just;
-    frr-agent = fancy.rustOverrides (self.callPackage ./nix/frr-agent {
-      rev = versions.frr-agent.rev;
-      hash = versions.frr-agent.hash;
-    });
+
+    # TODO: this really should be compiled with the same rustflags as the rest of the dataplane
```

Comment:
this comment is not a formatting change.  Will fix

Reply:
Found it too, but I decided that this was acceptable if it was the only change of content :sweat_smile: